Branch-SDK/src/io/branch/referral/RemoteInterface.java

```diff
@@ -156,6 +158,16 @@ private ServerResponse make_restful_get(String baseUrl, String tag, int timeout,
             timeout = DEFAULT_TIMEOUT;
         }
         if (addCommonParams(getParameters, retryNumber)) {
+            if (params != null) {
```

Comment:
Why add this in here and not in the ServerRequest getRequestPath of the subclass for the request? That's what is done with other GET requests.